docs/contributing.rst

```diff
@@ -1,5 +1,10 @@
-Contributing to py-evm
-----------------------
+Contributing
+------------
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. If you get stuck, feel free to reach for help in our `Gitter channel <https://gitter.im/ethereum/py-evm>`_.
+
+Setting the stage
+~~~~~~~~~~~~~~~~~
```

Comment:
Maybe a link to https://github.com/pipermerriam/ethereum-dev-tactical-manual/ is appropriate in this section?

Reply:
Sure, I agree. Some pages are more relevant than others to external contributors. This part seems the most relevant to them: https://github.com/pipermerriam/ethereum-dev-tactical-manual/blob/master/style-guide.md

Reply:
Thanks for bringing up the style guide. In fact, I re-read it again carefully and noticed a couple of things I did wrong on past PRs :+1: 

I'll make sure to link to it.